### PR TITLE
RVM-946 Added GetInstalledRuntimes API

### PR DESCRIPTION
**Pull request by Ben-Drebing**
Added API for user to query the RVM for the versions of all the runtimes installed.
Compatible with RVM 5.3+

fin.system.getInstalledRuntimes( )

Is linked to [this js-adapter PR](https://github.com/HadoukenIO/js-adapter/pull/307)

### CODE REVIEW

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@ben-drebing). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

File: src/browser/api_protocol/api_handlers/system.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export const SystemApiMap: APIHandlerMap = {
     'get-focused-window': getFocusedWindow,
     'get-focused-external-window': getFocusedExternalWindow,
     'get-host-specs': { apiFunc: getHostSpecs, apiPath: '.getHostSpecs' },
+    'get-installed-runtimes': getInstalledRuntimes,
```
Comment (datamadic) [inline src/browser/api_protocol/api_handlers/system.ts]: @rdepena does this need to be be able to be blocked via api-security?

Comment (datamadic) [inline src/browser/api_protocol/api_handlers/system.ts]: Can we style this one similar to `'get-host-specs': { apiFunc: getHostSpecs, apiPath: '.getHostSpecs' },` above it? The reason, while unfortunately not obvious, is that by doing so we can control it via group policy security settings.

File: src/browser/api/system.js [hunk 2]
```diff
@@ -301,6 +301,22 @@ export const System = {
             screenSaver: state.isScreenSaverRunning(),
         }, theme);
     },
+    getInstalledRuntimes: function(identity, callback, errorCallback) {
+        let getInstalledRuntimesOpts = {
+            sourceUrl: coreState.getConfigUrlByUuid(identity.uuid)
+        };
```
Comment (licui3936) [inline src/browser/api/system.js]: Do you need to add 'uuid' to this option?

File: src/browser/rvm/rvm_message_bus.ts [hunk 3]
```diff
@@ -398,7 +404,17 @@ export class RVMMessageBus extends EventEmitter  {
             }
         });
     }
-}
 
+    public getInstalledRuntimes(opts: GetInstalledRuntimesOptions, callback: (x: any) => any = () => undefined) {
```
Comment (rdepena) [inline src/browser/rvm/rvm_message_bus.ts]: return void instead of undefined.

**Pull request merged**